backend/tessend/tessend.go

```diff
@@ -0,0 +1,305 @@
+package tessend
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/coreos/etcd/clientv3"
+	v2 "github.com/sensu/sensu-go/api/core/v2"
+	"github.com/sensu/sensu-go/backend/ringv2"
+	"github.com/sensu/sensu-go/backend/store"
+	"github.com/sensu/sensu-go/backend/store/etcd"
+	"github.com/sensu/sensu-go/version"
+	"github.com/sirupsen/logrus"
+)
+
+const tessenURL = "https://tessen.sensu.io/v2/data"
+
+// Tessend is the tessen daemon
+type Tessend struct {
+	interval  uint32
```

Comment:
Why not make `interval` a `time.Duration`?

Reply:
My original decision was because `interval` is smaller and simpler than `time.Duration`. If you feel strongly that it should be `time.Duration` I can change it.